octreelib/ransac/cuda_ransac.py

```diff
@@ -0,0 +1,158 @@
+import numpy as np
+import numpy.typing as npt
+import numba as nb
+from numba import cuda
+from numba.cuda.random import create_xoroshiro128p_states
+
+from octreelib.internal import PointCloud
+from octreelib.ransac.initial_points_config import N_INITIAL_POINTS
+from octreelib.ransac.util import (
+    generate_random_indices,
+    get_plane_from_points,
+    measure_distance,
+)
+
+__all__ = ["CudaRansac"]
+
+
+N_CUDA_THREADS = 1024
+
+
+@cuda.jit
+def ransac_kernel(
```

Comment:
Why is this a separate function with class?

Reply:
made the kernel a staticmetod of `CudaRansac`